easybuild/easyconfigs/s/Sniffles/Sniffles-2.0.7-foss-2022a.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'PythonBundle'
+
+name = 'Sniffles'
+version = '2.0.7'
+
+homepage = 'https://github.com/fritzsedlazeck/Sniffles'
+description = """A fast structural variant caller for long-read sequencing,
+ Sniffles2 accurately detect SVs on germline, somatic and population-level for PacBio and Oxford Nanopore read data."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('Pysam', '0.20.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    (name, version, {
+        'modulename': '%(namelower)s',
```

Comment:
This shouldn't be needed, using the lowercase software name for the `import` check is the default